### ISSUE TITLE
Policy for IDP_DISCOVERY fails validation

### ISSUE THREAD

**Issue by nickerzb** [okta/terraform-provider-okta#232]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
Terraform Cloud (v.0.13.5)

### Affected Resource(s)
Okta Terraform Config -> data: Okta Default Policy of type IDP_DISCOVERY 

- okta_default_policy

### Terraform Configuration Files
```
data "okta_default_policy" "idp" {
  type = "IDP_DISCOVERY"
}
```

### Debug Output
```
Error: expected type to be one of [OKTA_SIGN_ON PASSWORD MFA_ENROLL OAUTH_AUTHORIZATION_POLICY], got IDP_DISCOVERY
```

Policy API for Okta clearly calls out that this type is allowed.
https://developer.okta.com/docs/reference/api/policy/#policy-object

**Comment by bogdanprodan-okta** [comment]
Hi @nickerzb ! I've merged your PR and it's now available in v.3.7.0 of the provider. Thanks again for fixing this issue!

**Comment by nickerzb** [comment]
https://github.com/oktadeveloper/terraform-provider-okta/pull/244

Found another issue.  I can't reference that default idp discovery policy due to the default name being different on this policy type (for some reason).  That should probably be consistent by default in the tenant, but I can't control that.

**Comment by bogdanprodan-okta** [comment]
Hi @nickerzb! This issue should be fixed in v3.7.3. If the error still exists, please reopen this issue. Thanks!
